### PR TITLE
[8.6.0] Fixes up `min` classifier handling

### PR DESCRIPTION
**Pull request by Kortanul**
`min` is a classifier, NOT part of the package name.

### CODE REVIEW

File: forgerock-ui-commons/pom.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- ~ DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
      ~
-     ~ Copyright (c) 2011-2013 ForgeRock AS. All Rights Reserved
+     ~ Copyright (c) 2011-2013 ForgeRock AS. All Rights Reserved.
```
Comment (siepkes) [inline forgerock-ui-commons/pom.xml]: Don't know if we need to touch this. ForgeRock is really sloppy with their copyright notices. For example [here](https://github.com/WrenSecurity/wrensec-parent/blob/fe797dddcd184514ed2e7c9c679b3113ca375ca7/pom.xml) they use `Copyright 2011-2015 ForgeRock AS.`. So we might just be better of not touching theirs.

Comment (Kortanul) [reply forgerock-ui-commons/pom.xml]: you're being too picky. but I'll change it back.

**Comment by Kortanul** [comment]
back to you, @siepkes

**Comment by Kortanul** [comment]
actually, I'm going to run with this for the 8.6 branch without the PR, since I need to make these fixes across several branches. but I'll still have you review the changes we use for the `master` branch. that way I can get us up and running without PRing changes to like 12 versions.

**Pull request closed without merging**